alf/examples/ddpg_pendulum.gin

```diff
@@ -0,0 +1,42 @@
+
```

Comment:
also add a picture of training curve

Reply:
red one represent for evaluate and blue for traning